tests/tests_integration/tests_diff_version/tests_common/test_commands.py

```diff
@@ -0,0 +1,29 @@
+from th2_data_services import Data
+
+def test_break_iteration_get_events(demo_events_from_data_source: Data):
+    iterated_one_item = False
+    for _ in demo_events_from_data_source:
+        if iterated_one_item:
+            break
+        iterated_one_item = True
+
+    count = 0
+
+    for _ in demo_events_from_data_source:
+        count += 1
+
+    assert count == 49 # Length of demo_events_from_data_source
+
+def test_break_iteration_get_messages(demo_messages_from_data_source: Data):
```

Comment:
Honestly, test for events and messages looks like the same. 
Is it possible to combine them?